src/test/kotlin/racingcar/domain/CarRaceTest.kt

```diff
@@ -0,0 +1,84 @@
+package racingcar.domain
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import racingcar.domain.Location.Companion.initialLocation
+
+class CarRaceTest {
+    @Test
+    fun `레이스 참가 차량 수를 입력받을 수 있다`() {
+        // given
+        val tCarCount = 3L
+        val tMaxRaceRound = 5L
+        val sut = CarRace(carRaceProperties(tCarCount, tMaxRaceRound, alwaysGoForwardRaceMoveRule()))
```

Comment:
테스트 클래스에서 작성된 만큼, 테스트를 나타내는 `t`나 `sut`라는 이름보단 의미를 명확히 드러내주는 이름으로 표현해주시면 테스트를 보는 사람이 조금 더 수월하게 읽을 수 있을 것 같아요!
```suggestion
         val numberOfCar = 3L
         val maxRaceRound = 5L
         val carRace = CarRace
```